### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by astroboyReloaded**
## For this section we followed the next steps: 📑 

- Add `#contact` + `<form>` **html** elements. 🔨
- Add corresponding **attributes** of `action`, `method`, `type`, `maxlength`, and `required`.  🔧 
- Add **CSS** file with `#contact {display: flex;}` and style to all `<form>` elements following the **Figma** design. 🎨 
- Connect `<form>` to **Formspree**. 🔌 
- Run linters. ✔️ 

😎

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -208,7 +209,42 @@ <h2>Skills</h2>
...
+            name="msg"
+            id="msg"
+            class="poppins"
+            placeholder="Enter your message here..."
+            maxlength="500"
+            required
+          ></textarea>
+          <button type="submit" class="btnBlue submit">
+            Start collaboration
+          </button>
```
Comment (DuaneDave) [inline index.html:245]: Good job implementing the form 👍, everything works as expected. However, according to the requirement the various state of the button should be styled to make the button more interactive and to signal to the user what state the button is in. With this in mind, I suggest that you implement styling for the hover` and  `active` states for the button.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @astroboyReloaded and @Vishal30M 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good PR summary.
✔️ The read me is very professional
✔️ Sleek design and implementation of validations.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (astroboyReloaded) [reply index.html:245]: **Thanks for your comment, @DuaneDave!**
On that topic, I want to point out that all buttons do have styling for `:focus` `:hover`, and `:active` states. It is in the rules for the `.btnBlue` at the **beginning** of the `styles.css` file; also you can see that just by hovering over the button. But I will add the `:visited` style also, now that you mention it.

✌️

Comment (DuaneDave) [reply index.html:245]: Yeah I noticed that, but I am speaking specifically about the submit button. Thanks for reaching out to clarify.

**Review by cwaku — APPROVED** [review]
Hi @team

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**